moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -77,175 +83,363 @@ ServoParameters::setParametersCallback(std::vector<rclcpp::Parameter> parameters
   return result;
 }
 
-ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcpp::Node::SharedPtr& node,
-                                                                     const rclcpp::Logger& logger,
-                                                                     std::string ns, /* = "moveit_servo"*/
-                                                                     bool dynamic_parameters /* = true */)
+void ServoParameters::declare(const std::string& ns,
+                              const rclcpp::node_interfaces::NodeParametersInterface::SharedPtr& node_parameters)
 {
-  auto parameters = std::shared_ptr<ServoParameters>(new ServoParameters(logger, ns));
+  using rclcpp::ParameterValue;
+  using rclcpp::ParameterType::PARAMETER_BOOL;
+  using rclcpp::ParameterType::PARAMETER_DOUBLE;
+  using rclcpp::ParameterType::PARAMETER_INTEGER;
+  using rclcpp::ParameterType::PARAMETER_STRING;
+  auto parameters = ServoParameters{};
 
-  // Get the parameters (organized same order as YAML file)
-  declareOrGetParam<bool>(parameters->use_gazebo, ns + ".use_gazebo", node, logger);
-  declareOrGetParam<std::string>(parameters->status_topic, ns + ".status_topic", node, logger);
+  // ROS Parameters
+  node_parameters->declare_parameter(
+      ns + ".use_gazebo", ParameterValue{ parameters.use_gazebo },
+      ParameterDescriptorBuilder{}
+          .type(PARAMETER_BOOL)
+          .description("Whether the robot is started in a Gazebo simulation environment"));
+  node_parameters->declare_parameter(
+      ns + ".status_topic", ParameterValue{ parameters.status_topic },
+      ParameterDescriptorBuilder{}.type(PARAMETER_STRING).description("Publish status to this topic"));
+  // Properties of incoming commands
+  node_parameters->declare_parameter(
+      ns + ".cartesian_command_in_topic", rclcpp::ParameterValue{ parameters.cartesian_command_in_topic },
```

Comment:
You don't need the rclcpp namespace for ParameterValue here because of the using statement.